scripts/characters/animal.gd

```diff
@@ -49,26 +49,22 @@ func throw(throwing_position: Vector3) -> void:
 	state = State.THROWN
 	is_thrown = true
 	
-func _physics_process(delta):
+func _physics_process(delta: float) -> void:
```

Comment:
what did types ever do to you 🥺 👉 👈

Reply:
Mostly just me being me and liking all my types visible if possible, but also GDScript runs faster when you strongly type it. Don't know if this directly applies to function arguments like this which are already typed by the function signature itself though.
https://godotengine.org/article/gdscript-progress-report-typed-instructions/#how-faster-is-it